docker-compose.yml

```diff
@@ -5,69 +5,74 @@ services:
       network: host
       context: ./
       args:
-        CONFIG: src/Lachain.Console/config0.json
-        WALLET: src/Lachain.Console/wallet0.json
+        CONFIG: src/Lachain.Console/poc/config01.json
+        WALLET: src/Lachain.Console/poc/wallet01.json
+        PRVH: src/Lachain.Console/poc/prv_h01.txt
       dockerfile: Dockerfile
     stdin_open: true
     environment:
-      - LOG_LEVEL=Trace
+      - LOG_LEVEL=Debug
     tty: true
-#    ports:
-#    - "5050:5050"
-#    - "7071:7070"
+    # ports:
+    #   - "7070:7070"
+    #   - "5050:5050"
+    # ports:
+    #   - "5050:5050"
+    #   - "7071:7070"
     network_mode: "host"
-    restart: always
+    restart: "no"
```

Comment:
I don't sure,  but looks like this property is used by cloud nodes too (for example,  to restart them after keys rescan)

Reply:
this is for in case if the container gets closed by any error, it will restart automatically...
Reverted back to `always`